### ISSUE TITLE
How to pretrain model on MS-COCO?

### ISSUE THREAD

**Issue by yifan254** [TheLegendAli/DeepLab-Context#25]
I'm the new to semantic segmentation, the deeplab paper shows that pre-train models on coco dataset can much improve the results, but the coco dataset just give the instance level labels, how to transfer data to satisfy the category level requirement, or anyone can give me some examples on how to pre-train models on coco datasets?
Thanks!

**Comment by yifan254** [comment]
I find a dataset called COCO-Stuff, the images are from COCO datasets and labeled with dense mark.
paper : https://arxiv.org/abs/1612.03716
datasets: https://github.com/nightrome/cocostuff
Hope this is helpful!

**Comment by ptirupat** [comment]
@yifan254 :Hi, Do you have any idea how to handle overlapping segments in the MS-COCO ground truth while training the network?
